packages/react/test/specs/behaviors/testDefinitions.ts

```diff
@@ -558,7 +558,7 @@ definitions.push({
   regexp: /Triggers '(\w+)' action with '(\w+)' on '([\w-]+)', when has an opened subtree\./g,
   testMethod: (parameters: TestMethod) => {
     const [action, key, elementToPerformAction] = [...parameters.props]
-    const propertyOpenedSubtree = { open: true, items: [{ a: 1 }] }
+    const propertyOpenedSubtree = { open: true, items: [{ a: 1 }], siblings: [], hasSubtree: true }
```

Comment:
```suggestion
    const propertyOpenedSubtree = { open: true, hasSubtree: true }
```

I think that is not required anymore

Reply:
still required as it will be used for the Hierarchical tree tests.